Makefile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.PHONY: all rbdplugin
+.PHONY: all rbdplugin cephfsplugin
```

Comment:
This is not a typo, but just a very small fix so I added it to this PR. If this PR should be for typo fix only or this change is not necessary, please let me know. I will update.